test/python/dagcircuit/test_dagcircuit.py

```diff
@@ -410,18 +410,6 @@ def test_remove_idle_clbit(self):
         self.assert_cregs_equal(self.original_cregs)
         self.assert_clbits_equal(self.original_clbits, excluding={self.individual_clbit})
 
-    def test_copy_circuit_metadata(self):
```

Comment:
I'm not sure if this tests should be removed.

Reply:
I think this test should be renamed to `test_copy_empty_like` as it tests `copy_empty_like` method instead of `_copy_circuit_metdata`.

Reply:
I agree.